dts/arm/nxp/nxp_imx8ml_m7.dtsi

```diff
@@ -188,3 +198,168 @@
 &nvic {
 	arm,num-irq-priority-bits = <4>;
 };
+
+
+/*
+ * GPIO pinmux options. These options define the pinmux settings
+ * for GPIO ports on the package, so that the GPIO driver can
+ * select GPIO mux options during GPIO configuration.
+ */
+
+&gpio1{
```

Comment:
For all the imx6/7/8 can this type of information be placed in a file without the core suffix (_m4, m7) since it's the same information for all cores?